data/bangs.json

```diff
@@ -107195,5 +107195,13 @@
     "u": "https://source.chromium.org/search?q={{{s}}}&ss=chromium/chromium/src",
     "c": "Tech",
     "sc": "Programming"
+  },
+  {
+    "s": "Discu.eu",
+    "d": "discu.eu",
+    "t": "discu",
+    "u": "https://discu.eu/?q={{{s}}}",
+    "c": "News",
+    "sc": "Social news/links"
```

Comment:
Can you add this bang somewhere in the middle? adding bangs exactly to the bottom tends to create merge conflicts